CheckValues/Results.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Text;
+using WaterLog_Backend;
+using WebApplication1;
+using WaterLog_Backend.Models;
+using WaterLog_Backend.Controllers;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CheckValues
+{
+    public class Results
+    {
+
+        public MonitorsController getController()
+        {
+            var builder = new ConfigurationBuilder();
+            builder.AddUserSecrets<Startup>();
+            var config = builder.Build();
+            //string mySecret = config["Localhost:ConnectionString"];
+
+            var optionsBuilder = new DbContextOptionsBuilder<DatabaseContext>();
+            optionsBuilder.UseSqlServer("Server = localhost; Database = waterlog; User Id = test; Password = test123");
+            //optionsBuilder.UseSqlServer(mySecret);
+            DatabaseContext _context = new DatabaseContext(optionsBuilder.Options);
+
+            MonitorsController _controller = new MonitorsController(_context, config);
+            return _controller;
+
+        }
+
+        public void item()
+        {
+
```

Comment:
Please clean up the white space in all the non-generated files.

Reply:
done